src/components/Button/index.scss

```diff
@@ -0,0 +1,113 @@
+.customButton {
+    transition: 0.3s;
+    outline: none;
+    box-sizing: border-box;
+    background: none;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+
+    span {
+        margin: 0 16px;
+    }
+
+    &_primary {
```

Comment:
Hey! We use '--' this sign for modificators, it was mentioned in link with Bem info

Reply:
ok